sal_interfaces/Electrometer/Electrometer_Events.xml

```diff
@@ -329,21 +324,21 @@ Protocols are those supported by the cURL library.</Description>
     </item>
     <item>
         <EFDB_Name>xonxoff</EFDB_Name>
-        <Description></Description>
+        <Description>Software flow control</Description>
```

Comment:
no idea what this is...

Reply:
so at a minimum you should say that and how to find out.

Reply:
I mentioned this in the description of this topic: This configuration needs to be the same as the one set in the Electrometer device. For proper configuration please read the reference manual 6517B-901-01 Rev. B / June 2009 section Selecting interface parameters